site/_docs/history.md

```diff
@@ -56,20 +132,221 @@ other software versions as specified in gradle.properties.
 #### New features
 {: #new-features-1-33-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-2884">CALCITE-2884</a>]
+  Implement JSON_INSERT, JSON_REPLACE, JSON_SET
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5105">CALCITE-5105</a>]
+  Add MEASURE type and AGGREGATE aggregate function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5155">CALCITE-5155</a>]
+  Custom time frames
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5283">CALCITE-5283</a>]
+  Add ARG_MIN, ARG_MAX (aka MIN_BY, MAX_BY) aggregate functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5362">CALCITE-5362</a>]
+  Implement geometry measurement functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5393">CALCITE-5393</a>]
+  VALUE as an synonym for VALUES keyword (enabled in MySQL conformance)
+
 #### Dependency version upgrade
 {: #dependency-1-33-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5341">CALCITE-5341</a>]
+  Upgrade Calcite to Avatica 1.23.0
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5351">CALCITE-5351</a>]
+  Bump jackson to 2.13.4 and jackson databind to 2.13.4.2 to avoid CVEs
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5356">CALCITE-5356</a>]
+  Update junit4 to 4.13.2 and junit5 to 5.9.1
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5374">CALCITE-5374</a>]
+  Upgrade jackson version to 2.14.0
+
 #### Bug-fixes, API changes and minor enhancements
 {: #fixes-1-33-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4186">CALCITE-4186</a>]
+  Add ST_CoveredBy spatial function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4351">CALCITE-4351</a>]
+  RelMdUtil#numDistinctVals always returns 0 for large inputs
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4632">CALCITE-4632</a>]
+  Exception in RelToSqlConverter: "Cannot convert x to DECIMAL(n, m) due to overflow"
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4804">CALCITE-4804</a>]
+  Support Snapshot operator serialization and deserialization
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4972">CALCITE-4972</a>]
+  Subfields of array columns containing structs are not qualified in getFieldOrigins
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4982">CALCITE-4982</a>]
+  Do not push 'cast to not null' through Join in ProjectJoinTransposeRule
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5127">CALCITE-5127</a>]
```

Comment:
How about we move CALCITE-5127 to the "new features" section? 

CC @zabetak cause you are the original reviewer of this feature.

Reply:
I agree this is a major change.

Reply:
done